client/redux-store.html

```diff
@@ -127,5 +131,18 @@
         return parsedState;
     }
 
+    function routeChanged(state, action) {
+        let n = {
+            currentRoute: action.route,
+            previousRouteName: state.currentRoute.name
+        };
+
+        if (state.currentRoute.name === 'DONATION-SUCCESS' || state.currentRoute.name !== 'DONATE-NOW') {
+            n.donationSuccessData = {};
```

Comment:
if the point is to clear `donationSuccessData` when navigating away from the `DONATION-SUCCESS` page, all we need to do is dispatch a `CLEAR-DONATION-SUCCESS-DATA` action inside `detached()` in `sc-donate-success.html`.

I'd prefer not to go down this route here.

Reply:
But what if we close the browser when we are on donation-success page. the `CLEAR-DONATION-SUCCESS-DATA` will not be dispatched

Reply:
take a look on line 125 in `redux-store.html`. You can reset the state there when loading it from `localStorage`.